helpers/GenerateResourcesAndImage.ps1

```diff
@@ -212,11 +212,11 @@ Function GenerateResourcesAndImage {
     } else {
         # Parametrized Authentication via given service principal: The service principal with the data provided via the command line
         # is used for all authentication purposes.
-        $spAppId = $AzClientId
-        $spClientId = $AzClientId
+        $spAppId = $AzureClientId
+        $spClientId = $AzureClientId
         $credentials = $AzureAppCred
-        $ServicePrincipalClientSecret = $AzClientSecret
-        $tenantId = $AzTenant
+        $ServicePrincipalClientSecret = $AzureClientSecret
+        $tenantId = $AzureTenant
```

Comment:
$AzureTenantId rather than $AzureTenant?

Reply:
Good point; thank you for the feedback. I have updated it.